tests/functional/syntax/test_import.py

```diff
@@ -0,0 +1,134 @@
+import pytest
+
+from vyper import compiler
+from vyper.exceptions import ModuleNotFound
+
+
+def test_implicitly_relative_import_crashes(make_input_bundle):
+    top = """
+import subdir0.lib0 as lib0
+@external
+def foo():
+    lib0.foo()
+    """
+
+    lib0 = """
+import subdir1.lib1 as lib1
+def foo():
+    lib1.foo()
+    """
+
+    lib1 = """
+def foo():
+    pass
+    """
+
+    input_bundle = make_input_bundle(
+        {"top.vy": top, "subdir0/lib0.vy": lib0, "subdir0/subdir1/lib1.vy": lib1}
+    )
+
+    with pytest.raises(ModuleNotFound):
```

Comment:
i think these might not be raising for the right reason -- if resolved_path is not set in compile_code, ModuleNotFound will be raised anyways